src/Templates/Laravel/Factory.txt

```diff
@@ -7,8 +7,5 @@
 
 $factory->define(_namespace_model_\_camel_case_::class, function (Faker\Generator $faker) {
     return [
-
-        // _camel_case_ table data
```

Comment:
Can we correct these, the system will place the schema in here and with the closing ] on the same line it may break things